tofu/mag/test_magFieldLines.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+# Also if needed: retab
+'''
+    TEST magFieldLines
+'''
+from __future__ import (unicode_literals, absolute_import,  \
+                        print_function, division)
+import doctest
+import os
+import sys
+
+#print('path 1 =', sys.path)
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
Please make sure any change to the system path is exceptional and temporary
The system path should not be changed without the user knowing, unless it is temporary (i.e.: only in for a limited time inside the file)

=> please restore the original system path as soon as possible after the tests are run

=> Actually, this is not necessary, all unit tests should be placed in tofu/tests/

=> Temporary fix: restore original system path in the file, I validate the PR, then we open a new issue to move all unit tests to the proper repo (we will create a dedicated tofu/tests/tests09_mag/ repo, and get the template from other unit tests